sdcm/utils/common.py

```diff
@@ -1015,3 +1015,14 @@ def get_branched_ami(ami_version, region_name):
         return amis
     else:
         return amis[:1]
+
+
+def measure_time(func):
+    @wraps(func)
```

Comment:
can you add a docstring to this function, to explain what it's doing ? (see `log_run_info`)

Reply:
done